Login slug (#8354)
fishtown-analytics/docs.getdbt.com: website/snippets/login-slug.md

> To complete this section, you will need your login URL slug. This slug controls the URL where users on your account can log into your application. Login URL slugs are automatically set by dbt and can't be altered. It should contain only letters, numbers, and dashes.

Author: ```suggestion
To complete this section, you will need your login URL slug. This slug controls the URL where users on your account can log into your application. dbt automatically generates login URL slugs, which can't be altered. It will contain only letters, numbers, and dashes.
```

> For example, the login URL slug for dbt Labs would be `dbt-labs`.

Author: ```suggestion
For example, the login URL slug for dbt Labs would look something like `dbt-labs-afk123`.
```
